sonar-db/src/main/java/org/sonar/db/issue/IssueFilterDao.java

```diff
@@ -72,27 +81,45 @@ public IssueFilterDto selectProvidedFilterByName(String name) {
     }
   }
 
+  /**
+   * @deprecated since 5.2 use {@link #selectSharedFilters(DbSession)}
+   */
+  @Deprecated
   public List<IssueFilterDto> selectSharedFilters() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)